src/DependencyInjection/SonataMediaExtension.php

```diff
@@ -548,6 +548,16 @@ private function configureAdapters(ContainerBuilder $container, array $config):
 
     private function configureResizers(ContainerBuilder $container, array $config): void
     {
+        if ($container->hasParameter('sonata.media.resizer.crop.class')) {
+            $class = $container->getParameter('sonata.media.resizer.crop.class');
+            $definition = new Definition($class, [
+                new Reference('sonata.media.adapter.image.default'),
+                new Reference('sonata.media.metadata.proxy'),
+            ]);
+            $definition->addTag('sonata.media.resizer');
+            $container->setDefinition('sonata.media.resizer.crop', $definition);
```

Comment:
Can we create `sonata.media.resizer.base_crop` for default class and `sonata.media.resizer.crop` for service or class configured by users?